src/data/polynomial/basic.lean

```diff
@@ -261,19 +253,13 @@ by { rcases p, simp [support] }
 lemma card_support_eq_zero : p.support.card = 0 ↔ p = 0 :=
 by simp
 
-/-- `monomial s a` is the monomial `a * X^s` -/
-def monomial (n : ℕ) : R →ₗ[R] R[X] :=
-{ to_fun := monomial_fun n,
-  map_add' := by simp [monomial_fun],
-  map_smul' := by simp [monomial_fun, ←of_finsupp_smul] }
```

Comment:
Any reason for moving this?

Reply:
I did it originally to try and figure out whether it could be moved up as I had to redefine `1` while doing this. I'll move it down when tests succeed, as I don't want to trigger a full rebuild until I know stuff is okay